lib.js

```diff
@@ -18,7 +18,13 @@ var conversions = [
   {
     regex: /miles?|mi\.?/,
     target: " km",
-    conversion: x => x * 1.6093
+    conversion: x => x * 1.609344
+  },
+  // speed
+  {
+    regex: /mph/,
+    target: " kph",
```

Comment:
I'm afraid "kph" is another Americanism, the proper abbreviation¹ is "km/h".

(1: as per international usage, and SI standard)